jdaviz/tests/test_utils.py

```diff
@@ -24,50 +25,120 @@ def test_spec_sb_flux_conversion():
 
     # Float scalar pixel scale factor
     spec.meta["_pixel_scale_factor"] = 0.1
-    assert_allclose(flux_conversion(spec, values, u.Jy / u.sr, u.Jy), [1, 2, 3])
-    assert_allclose(flux_conversion(spec, values, u.Jy, u.Jy / u.sr), [100, 200, 300])
+    assert_allclose(flux_conversion(values, u.Jy / u.sr, u.Jy, spec), [1, 2, 3])
+    assert_allclose(flux_conversion(values, u.Jy, u.Jy / u.sr, spec), [100, 200, 300])
+
+    # complex translation Jy / sr -> erg / (Angstrom s cm2 sr)
+    targ = [2.99792458e-12, 1.49896229e-12, 9.99308193e-13] * (u.erg / (u.Angstrom * u.s * u.cm**2 * u.sr))  # noqa: E501

```

Comment:
How did you obtain the "truths"?

Reply:
I ran the logic in a notebook cell that did each step in the translation -> conversion -> conversion -> translation (and did this for each of the added tests).

For this specific case it was Jy / sr to Jy, and converting with the translation scale factor, then Jy to erg /(Angstrom *s * cm^2), using the spectral density equivalencies from the original spectrum, then, we do the translation from erg /(Angstrom *s * cm^2) to erg /(Angstrom *s * cm^2 * sr), which is using the same scale factor. 

So really it is a reproduction of the logic in the function, but done step by step in a notebook to validate that each value is what is expected. 

I would assume the main scientific conversion in question would be going from Jy to erg /(Angstrom *s * cm^2), and having scientific validation that proves these values are correct might be worth while to back up what astropy's spectral density equivalencies are correct (I assumed they were).